src/hotspot/share/gc/shenandoah/shenandoahVerifier.cpp

```diff
@@ -411,11 +411,12 @@ class ShenandoahGenerationStatsClosure : public ShenandoahHeapRegionClosure {
               "%s: generation (%s) used regions (" SIZE_FORMAT ") must equal regions that are in use (" SIZE_FORMAT ")",
               label, generation->name(), generation->used_regions(), stats.regions());
 
-    size_t capacity = generation->adjusted_capacity();
-    guarantee(stats.span() <= capacity,
-              "%s: generation (%s) size spanned by regions (" SIZE_FORMAT ") must not exceed current capacity (" SIZE_FORMAT "%s)",
-              label, generation->name(), stats.regions(),
-              byte_size_in_proper_unit(capacity), proper_unit_for_byte_size(capacity));
+// This check is disabled because of known issues with this feature. We expect this code to be updated by 05/2023.
```

Comment:
I understand the "resolved conversation."  Just want to add that this is still failing on the expand-on-demand branch, and I'm supposed to fix it there...

Reply:
Hmm... no. If this isn't fixed by `expand-on-demand` we need to dig deeper.